kubernetes/template/manifest.json

```diff
@@ -57,7 +57,7 @@

```

Comment:
Am I allowed to update the legal terms? I only changed ACS to AKS

Reply:
that should be OK.